react/src/components/lablupTalkativotUI/EndpointLLMChatCard.tsx

```diff
@@ -130,6 +149,24 @@ const EndpointLLMChatCard: React.FC<EndpointLLMChatCardProps> = ({
           />
         )
       }
+      inputMessage={isSynchronous ? synchronizedMessage : undefined}
+      onInputChange={(v) => {
+        if (!_.isUndefined(v)) {
```

Comment:
Why is this condition needed?

Reply:
to avoid type error 😇 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/7a91d40e-6ba3-4385-aa56-50c9e526816e.png)

Reply:
to avoid type error 😇 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/2HueYSdFvL8pOB5mgrUQ/7a91d40e-6ba3-4385-aa56-50c9e526816e.png)